cmd/client.go

```diff
@@ -25,7 +25,7 @@ type ClientConfig struct {
 	// currently configured in the client's connection should be used.
 	Namespace string
 
-	// Verbose logging.  By default logging output is kept to the bare minimum.
+	// Verbose logging.  By default, logging output is kept to the bare minimum.
```

Comment:
Personally, I would prefer to keep small changes like this to commits that also modify the file in some more meaningful way. In this case, adding a comma is the only change to the file. Minor nit - no need to change it, unless you're already making other updates.